facade/src/test/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerTest.java

```diff
@@ -117,6 +128,11 @@ public void initMock() {
                     .orElse(null);
         });
         when(artifactRepository.save(any())).thenAnswer(new RepositorSave(artifacts));
+
+        when(artifactRepository.withSha256AndDependantBuildRecordIdIn(anyString(), anySet()))
+                .thenReturn(Collections.emptyList());
```

Comment:
`By default, for all methods that return a value, a mock will return either null, a primitive/primitive wrapper value, or an empty collection, as appropriate. For example 0 for an int/Integer and false for a boolean/Boolean.`

Reply:
ok thanks so these 3 would not be necessary